packages/gestalt/src/RadioButton.css

```diff
@@ -1,75 +1,88 @@
 :root {
+  --border-sm: 6px;
+  --border-md: 8px;
+  --border: 2px;
+  --darkGray: #111;
+  --lightGray: #efefef;
   --size-sm: 16px;
   --size-md: 24px;
+  --solid: solid;
+  --translucidGray: rgba(0, 0, 0, 0.12);
 }
 
-.RadioButton {
+.radioButton {
   composes: block from "./Layout.css";
   composes: borderBox from "./Layout.css";
   composes: circle from "./Borders.css";
   composes: flex from "./Layout.css";
   composes: itemsCenter from "./Layout.css";
   composes: justifyCenter from "./Layout.css";
   composes: relative from "./Layout.css";
-  border: 1px solid #8e8e8e;
+  composes: sizeLg from "./Borders.css";
+  composes: solid from "./Borders.css";
+  composes: borderColorLightGray from "./Borders.css";
 }
 
-.RadioButtonSm {
-  height: var(--size-sm);
-  padding: 2px;
-  width: var(--size-sm);
+.radioButton:hover:not(.focused):not(.borderChecked) {
+  border-color: var(--translucidGray);
 }
 
-.RadioButtonMd {
-  height: var(--size-md);
-  padding: 3px;
-  width: var(--size-md);
+.borderChecked {
+  border-color: var(--darkGray);
 }
 
-.RadioButtonIsFocused {
-  composes: accessibilityOutlineFocus from "./Focus.css";
+.borderCheckedSm {
+  border-width: var(--border-sm);
 }
 
-.RadioButtonWhiteBg {
-  composes: whiteBg from "./Colors.css";
+.borderCheckedMd {
+  border-width: var(--border-md);
+}
+
+.borderDisabled {
+  composes: borderColorLightGray from "./Borders.css";
 }
 
-.RadioButtonLightGrayBg {
+.bgDisabled {
   composes: lightGrayBg from "./Colors.css";
 }
 
-.Input {
+.bgEnabled {
+  composes: whiteBg from "./Colors.css";
+}
+
+.input {
   composes: absolute from "./Layout.css";
   composes: m0 from "./Whitespace.css";
   appearance: none;
   opacity: 0;
   outline: 0;
 }
 
-.InputEnabled {
+.inputEnabled {
   composes: pointer from "./Cursor.css";
 }
 
-.InputSm {
+.inputSm {
```

Comment:
@AlbertCarreras Why do we need both `inputSm` and `sizeSm` when they have the exact same CSS? Can we pick one?

Reply:
DONE